### PR TITLE
Added making stuff happen post

### PR DESCRIPTION
**Pull request by robhuzzey**
### What does this PR do?
Too many todo's... here's how you make stuff happen

### How do i test?
- Download the branch, run ```jekyll serve```
- Visit http://localhost:4000/workflow/2015/02/24/making-stuff-happen/

#### Quality checklist (for PR author to complete BEFORE code review):
- [/] I've updated documentation with any changes to procedures.
- [/] I've tested this on mobile Android, Safari, Chrome for any visual changes.
- [x] I am ready for this to be code reviewed, merged and tested.

### Note
I was unable to get Jekyll working, too little time, will fix later, so if this could be reviewed with the reviewer checking for visuals that would be awesome, thanks.

### CODE REVIEW

File: _posts/2015-02-24-making-stuff-happen.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+3. **Tell somebody:** Just by sharing the idea, you are more likely to complete it because you would likely get feedback, good or bad, that will drive you towards achieving that goal.
+
+4. **Make a "do not do" list:** Things like Facebook, watching a TV show & even cooking a meal take up time. Get takeout / Turn off the TV / Block Facebook & use that time saved to work on your idea. If you really start to think about it, you waste a lot more time that you realise.
+
+5. **Get up an hour earlier:** You will be less likely to procrastinate in the morning (maybe after a coffee to get you going) and it's better for you than staying up an hour late.
+
+
+If you really can't do any of the above, there is a more extreme option…
+
+**Remove it:** Yep, just take an idea off your list! If you really can't bear to delegate it or it's been stagnating for ages, kill it off, all those extra ideas that clutter up your list are making the whole thing seem overwhealming. By removing a few (or at least one), you will find better clarity & focus to pick off the tasks that really matter to you.
```
Comment (Joezo) [inline _posts/2015-02-24-making-stuff-happen.md]: typo in `overwhealming`

File: _posts/2015-02-24-making-stuff-happen.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title: Making stuff happen
+categories: workflow
+author: Rob Huzzey
+tags: todo workflow lists
+comments: true
+---
+
+Making stuff happen
```
Comment (jodiedoubleday) [inline _posts/2015-02-24-making-stuff-happen.md]: This title is not needed as the title is already set in the yaml front matter

screen shot 2015-02-24 at 11 16 36

**Comment by jackdcrawford** [comment]
Looks good on mine @robhuzzey

**Comment by jodiedoubleday** [comment]
:+1: from me

**Pull request merged**